src/olympia/users/models.py

```diff
@@ -274,7 +275,7 @@ def picture_path_original(self):
     def picture_url(self):
         from olympia.amo.templatetags.jinja_helpers import user_media_url
         if not self.picture_type:
-            return settings.STATIC_URL + '/img/zamboni/anon_user.png'
```

Comment:
you can't change this in the `Addon` model because it's still used in legacy frontend views.  You'll need to handle in `BaseUserSerializer` instead